spaceinvaders.py

```diff
@@ -211,12 +214,11 @@ def update(self, keys, *args):
 
 
 class Mystery(sprite.Sprite):
-    def __init__(self):
-        sprite.Sprite.__init__(self)
-        self.image = IMAGES['mystery']
-        self.image = transform.scale(self.image, (75, 35))
+    def __init__(self, *groups):
```

Comment:
What is `*groups`?

Reply:
See [prev comment](https://github.com/leerob/Space_Invaders/pull/23#discussion_r250292373). Also, an adding to groups in `init` is little bit shortie:
```
Mystery(self.allSprites, self.mysteryGroup)
# Instead of
newShip = Mystery()
self.allSprites.add(newShip)
self.mysteryGroup.add(newShip)
...

Reply:
A little bit of a feng shui in the 1e87cac `Replace unobvious Sprite(*groups) w Sprite.add(*groups)`.